packages/vitest/src/node/cli/cli-config.ts

```diff
@@ -602,24 +696,32 @@ export const cliOptionsConfig: VitestCLIOptions = {
         argument: '',
         subcommands: {
           interval: {
-            description: 'Poll interval in milliseconds for `expect.poll()` assertions (default: `50`)',
+            description:
+              'Poll interval in milliseconds for `expect.poll()` assertions (default: `50`)',
             argument: '<interval>',
           },
           timeout: {
-            description: 'Poll timeout in milliseconds for `expect.poll()` assertions (default: `1000`)',
+            description:
+              'Poll timeout in milliseconds for `expect.poll()` assertions (default: `1000`)',
             argument: '<timeout>',
           },
         },
         transform(value) {
-          if (typeof value !== 'object')
-            throw new Error(`Unexpected value for --expect.poll: ${value}. If you need to configure timeout, use --expect.poll.timeout=<timeout>`)
+          if (typeof value !== 'object') {
+            throw new TypeError(
```

Comment:
Was the changed from `Error` to `TypeError` here intentional? Tests for this started to fail now.

Reply:
Not intentional, but sounds right. I think we have a eslint rule for that